shotover-proxy/src/message/mod.rs

```diff
@@ -345,26 +350,33 @@ impl Value {
             match spec.col_type.id {
                 ColType::Ascii => Value::Strings(decode_ascii(actual_bytes).unwrap()),
                 ColType::Bigint => Value::Integer(decode_bigint(actual_bytes).unwrap()),
-                ColType::Blob => Value::Bytes(Bytes::copy_from_slice(actual_bytes)),
                 ColType::Boolean => Value::Boolean(decode_boolean(actual_bytes).unwrap()),
-                ColType::Counter => Value::Integer(decode_int(actual_bytes).unwrap() as i64),
-                ColType::Decimal => unimplemented!("We dont have a decimal type yet"),
+                ColType::Counter | ColType::Int => {
+                    Value::Integer(decode_int(actual_bytes).unwrap() as i64)
+                }
+                ColType::Decimal => todo!(),
                 ColType::Double => Value::Float(decode_double(actual_bytes).unwrap()),
                 ColType::Float => Value::Float(decode_float(actual_bytes).unwrap() as f64),
-                ColType::Int => Value::Integer(decode_int(actual_bytes).unwrap() as i64),
-                ColType::Uuid => Value::Bytes(Bytes::copy_from_slice(actual_bytes)),
                 ColType::Varchar => Value::Strings(decode_varchar(actual_bytes).unwrap()),
-                ColType::Varint => unimplemented!("We dont have a varint type yet"),
-                ColType::Timeuuid => Value::Bytes(Bytes::copy_from_slice(actual_bytes)),
+                ColType::Varint => todo!(),
```

Comment:
Do we make another Value variant for Decimal or store it in the int one?

Reply:
Let's look at what the Java implementation of Varint is and see if there is a stdlib one in Rust or if we need to implement our own / import a lib